src/core/query-builder.utils.test.ts

```diff
@@ -0,0 +1,111 @@
+import { expressionBuilderCallback, expressionReaderCallback, setQueryComputedFields } from "./query-builder.utils"
+
+jest.mock('./query-builder.constants.ts', () => ({
+  ...jest.requireActual('./query-builder.constants'),
+  ComputedDataFields: {
+    Obj1: 'obj1.prop1 || obj1.prop2',
```

Comment:
from my understanding , the agreement is not to abbreviate words .This was a feedback to me as well from @CiprianAnton